src/fairseq2/models/llama/builder.py

```diff
@@ -357,3 +358,12 @@ def create_llama_model(
         The data type of module parameters and buffers.
     """
     return LLaMABuilder(config, device=device, dtype=dtype).build_model()
+
+
+def get_llama_lora_config() -> LoRAConfig:
+    return LoRAConfig(
+        r=8,
+        alpha=16.0,
+        dropout_p=0.05,
+        keys=[".*decoder.layers.*.self_attn.*(q_proj|v_proj)$"],
```

Comment:
Just to double check, we do not fine-tune K projection, is that right?

Reply:
Yes, only q and v projections. This is the default setting from the original LoRA paper (https://arxiv.org/abs/2106.09685).

Though it might be worth noting that, recent papers have shown that fine-tuning only q&v projections might not be sufficient for larger models. The recent QLoRA paper (https://arxiv.org/abs/2305.14314) suggested fine-tuning all projection layers.